src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -275,8 +275,15 @@ export class SettingsEditor2 extends EditorPane {
 			SettingsEditor2.SUGGESTIONS.push(`@${LANGUAGE_SETTING_TAG}`);
 		}
 
-		extensionManagementService.getInstalled(ExtensionType.System).then(extensions => {
-			this.installedExtensionIds = extensions.map(extension => extension.identifier.id);
+		Promise.all([extensionManagementService.getInstalled(ExtensionType.System), extensionManagementService.getInstalled(ExtensionType.User)]).then(extensions => {
```

Comment:
You can get all extension without passing any type, eg: `extensionManagementService.getInstalled()`

Reply:
That greatly simplified the implementation. Added a commit to the PR.